CodingChallenges/CC_003_Snake_game/P5/sketch.js

```diff
@@ -29,23 +27,16 @@ function mousePressed() {
 
 function draw() {
   background(51);
-
-  if (s.eat(food)) {
+ if (s.eat(food)) {
```

Comment:
Might be wrong but it looks like a space on this line got deleted. Was that intentional?

Reply:
Yes. Those were just blank spaces

Reply:
Sorry I meant the that there was a space missing in the indentation. Added a commit to fix it. Sorry I probably didn't use the best wording.